app/views/content_partners/show.html.erb

```diff
@@ -28,5 +31,33 @@
 
   <%= render 'shared/pagination_below_table', items: @extraction_definitions %>
 <% else %>
-  <p>There are currently no extraction definitions for this content partner.</p>
+  <p>There are currently no extraction definitions for <%= @content_partner.name_in_database %>.</p>
+<% end %>
+
+<h2 class="float-start">Transformations</h2>
```

Comment:
Would it be good to have Transformations and Extraction Definitions in their own partials?  Might make it easier to follow what is going on.  Just an idea.

Reply:
Yeah that sounds good :)